schemas/spdx-schema.json

```diff
@@ -4,606 +4,95 @@
-          "type" : "string"
-        },
-        "externalDocumentRefs" : {
-          "description" : "Identify any external SPDX documents referenced within this SPDX document.",
-          "type" : "array",
-          "items" : {
-            "type" : "object",
-            "properties" : {
-              "externalDocumentId" : {
-                "description" : "externalDocumentId is a string containing letters, numbers, ., - and/or + which uniquely identifies an external document within this document.",
-                "type" : "string"
-              },
-              "checksum" : {
-                "type" : "object",
-                "properties" : {
-                  "algorithm" : {
-                    "description" : "Identifies the algorithm used to produce the subject Checksum. Currently, SHA-1 is the only supported algorithm. It is anticipated that other algorithms will be supported at a later time.",
-                    "type" : "string",
-                    "enum" : [ "SHA256", "SHA1", "SHA384", "MD2", "MD4", "SHA512", "MD6", "MD5", "SHA224" ]
-                  },
-                  "checksumValue" : {
-                    "description" : "The checksumValue property provides a lower case hexidecimal encoded digest value produced using a specific algorithm.",
-                    "type" : "string"
-                  }
-                },
-                "description" : "A Checksum is value that allows the contents of a file to be authenticated. Even small changes to the content of the file will change its checksum. This class allows the results of a variety of checksum and cryptographic message digest algorithms to be represented."
-              },
-              "spdxDocument" : {
```

Comment:
All I wanted was to change this item from spdxDocument to documentNamespace

Reply:
And this is buried inside the external reference property.  There's nothing for the namespace for this document